nwb_conversion_tools/baserecordingextractorinterface.py

```diff
@@ -75,14 +75,13 @@ def run_conversion(
       nwbfile: NWBFile,
       metadata: dict = None,
       stub_test: bool = False,
-      use_times: bool = False, 
-      write_as_lfp: bool = False,
-      save_path: PathType = None, 
+      use_times: bool = False,
+      save_path: OptionalPathType = None,
       overwrite: bool = False,
       buffer_mb: int = 500
     ):
         """
-        Primary function for converting recording extractor data to nwb.
+        Primary function for converting high-pass recording extractor data to nwb.
```

Comment:
not necessarily `high-pass`, right? More like raw data

Reply:
Is there any type of `RecordingExtractor` data that is not either high-pass or low-pass? If high-pass, this is the interface to inherit from and the `run_conversion` to use. If low-pass, then they should use the LFPDataInterface.

Note since these interfaces are specific to those in spikeextractors, it foregoes those types of raw (first level) data that are sampled at frequencies that could be considered low-pass, like ecog.

Correct me if I'm wrong though, I'm just basing it off what I've seen so far in spikeextractors.

Reply:
This is a minor comment on nomenclature, high-pass is filtered data, and most cases I believe the source data will be raw data (non-filtered).
High-pass filtered data is valuable for spike sorting for example, so maybe we would also like to provide an interface for that, storing in the `pynwb.ecephys.FilteredEphys` container.
we could think about generalizing the RecordingInterfaces to map ecephys traces like this:

raw data → acquisition
low-pass → pynwb.ecephys.LFP
high-pass (or any other filtered data) → pynwb.ecephys.FilteredEphys

Reply:
Ahh, I see - thanks! I'll adjust the docstring then and add the high-pass interface as a to-do (I know it can come up sometimes in the spikeinterfac pipelines)